docs/sources/datasources/prometheus.md

```diff
@@ -21,27 +21,29 @@ Grafana includes built-in support for Prometheus. This topic explains options, v
+| `Default`                       | Default data source that is pre-selected for new panels.                                                                                                                                                                                                                                                        |
+| `URL`                           | The URL of your Prometheus server, for example, `http://prometheus.example.org:9090`.                                                                                                                                                                                                                           |
+| `Access`                        | Only Server access mode is functional. If Server mode is already selected this option is hidden. Otherwise change to Server mode to prevent errors.                                                                                                                                                             |
+| `Basic Auth`                    | Enable basic authentication to the Prometheus data source.                                                                                                                                                                                                                                                      |
+| `User`                          | User name for basic authentication.                                                                                                                                                                                                                                                                             |
+| `Password`                      | Password for basic authentication.                                                                                                                                                                                                                                                                              |
+| `Manage alerts via Alerting UI` | Toggle whether to enable Alertmanager integration for this data source.                                                                                                                                                                                                                                         |
+| `Alertmanager data source`      | The [Alertmanager data source]({{< relref "./alertmanager/" >}}) to use for alert management.                                                                                                                                                                                                                   |
```

Comment:
We're going to remove this option because there is no way to check if it aligns with the actual Prometheus configuration https://github.com/grafana/grafana/pull/57369